src/sampleApiData.js

```diff
@@ -36,4 +36,41 @@ export const sampleApiData = {
   },
   "base": "USD",
   "date": "2021-01-14"
-}
\ No newline at end of file
+}
+
+export const currencyCodes = [
```

Comment:
I can see that this could be a problem if the api ever changes. But it works and I happy with this for today! 💯